core/src/dird/dird_conf.cc

```diff
@@ -115,8 +115,9 @@ static DeviceResource* res_dev;
 static UserResource* res_user;
 
 
+static std::string query_file_path
+    = std::string(PATH_BAREOS_SCRIPTDIR) + "/query.sql";
```

Comment:
as I'm just looking at this: did you know that you can just use `PATH_BAREOS_SCRIPTDIR "/query.sql"` as a string constant instead of creating an intermediate string?